pages/searching/time_frame_selector.rst

```diff
@@ -1,38 +1,96 @@
-Time frame selector
+Time Frame Selector
```

Comment:
I know this page existed before this PR, but the application uses the words "Time range" on-screen instead of "Time frame". I think the docs should be consistent with the words the app uses, and I don't see "frame" being used in the app.

What do you think? Does it make sense to replace all instances of "frame" in this document with "range"?

![image](https://user-images.githubusercontent.com/3423655/123155435-bded6280-d42d-11eb-9e9e-b72a79a1ed63.png)

Reply:
Totally agree with this. Finally getting to updating this.

Reply:
As discussed with @dulanism, this difference will be resolved later.